source/documentation/monitoring-an-app/how-to-use-cloudwatch-datasource.html.md.erb

```diff
@@ -0,0 +1,52 @@
+---
+title: Using the CloudWatch data source in Grafana 
+last_reviewed_on: 2020-02-14
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+### Introduction
+
+Cloud-platform team added [CloudWatch data source][CloudWatch-datasource] in Grafana, using this [CloudWatch data source][CloudWatch-datasource] you can build dashboards for your CloudWatch metrics.
+
+#### Using the CloudWatch data source in Grafana 
+
+Cloud-platform team created some default dashboards, you can access them [here][cp-dashboards] or you can build your own dashboards for your CloudWatch metrics, you can also do dynamic queries using dimension wildcards.
+
+Create a dashboard using CloudWatch data source:
+
+From the official & community built [dashboards][default-dashboards] for CloudWatch data source, select the AWS resource you want to create the dashboard. Download it as JSON and modify it as you see fit. Copy the JSON string into a `ConfigMap` according to the example below.
+
+```YAML
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: <my-dashboard>
+  namespace: <my-namespace>
+  labels:
+    grafana_dashboard: ""
+data:
+  my-dashboard.json: |
+    {
+      [ ... ]
+    }
+```
+
+Add `ConfigMap` above to your namespace folder in the [environments repo][env-repo]. Once the ConfigMap applied though the concourse pipeline, your dashboard should be visible in Grafana shortly.
```

Comment:
Add the `ConfigMap`...  (missing 'the')
Once the ConfigMap is applied ... (missing 'is')